configure.d/1_submit_bio.conf

```diff
@@ -35,26 +25,14 @@ apply() {
     case "$1" in
     "1")
 		add_function "
-	static inline blk_qc_t cas_submit_bio(int rw, struct bio *bio)
+	static inline void cas_submit_bio(int rw, struct bio *bio)
 	{
 		CAS_BIO_OP_FLAGS(bio) |= rw;
-		return submit_bio(bio);
+		submit_bio(bio);
 	}" ;;
+
```

Comment:
How about kernel versions where `submit_bio()` returns an error code?

Reply:
Most drivers never checked the return value from submit_bio even when submit_bio was declared non-void. It is safe to leave it as it is.

For example, in 4.18, submit_bio is declared as "`blk_qc_t submit_bio(struct bio *bio)`", yet none of the drivers in the ./drivers directory make any use of its return value.